libr/bin/bin.c

```diff
@@ -1401,7 +1401,7 @@ R_API RList *r_bin_get_mem(RBin *bin) {
 
 R_API int r_bin_is_big_endian(RBin *bin) {
 	RBinObject *o = r_bin_cur_object (bin);
-	return o? o->info->big_endian: false;
+	return o? o->info->big_endian: -1;
```

Comment:
ok

> On 03 May 2016, at 22:06, Damien Zammit <notifications@github.com> wrote:
> 
> In libr/bin/bin.c <https://github.com/radare/radare2/pull/4715#discussion_r61946516>:
> 
> > @@ -1401,7 +1401,7 @@ R_API RList *r_bin_get_mem(RBin *bin) {
> >  
> >  R_API int r_bin_is_big_endian(RBin *bin) {
> >  	RBinObject *o = r_bin_cur_object (bin);
> > -	return o? o->info->big_endian: false;
> > +	return o? o->info->big_endian: -1;
> No, if o is null, it means there was no binary detected so its a third case
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4715/files/820b97df9eb03df8e2490c097399b43bad1b8ae5#r61946516>